scripts/west_commands/runners/__init__.py

```diff
@@ -2,9 +2,14 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
+# stdlib
 import importlib
 import logging
 
+# exported
+from runners.core import MissingProgram as MissingProgram
+
+# used
```

Comment:
Not sure what to replace it with here. The advise from another warning was to make an "useless import alias" as a way to tell it was exported, and defining an `__all__` manually with all exports is more intrusive.

Reply:
Does https://github.com/zephyrproject-rtos/zephyr/pull/81810 help? I think re-exporting doesn't make a lot of sense.

Reply:
```
$ west flash
-- west flash: rebuilding
ninja: no work to do.
-- west flash: using runner ecpprog
init..
IDCODE: 0x010fb043 (LIFCL-33U)
NX Status Register: 0x0000110100000100
reset..
flash ID: 0xEF 0x80 0x18
file size: 170788
erase 64kB sector at 0x100000..
erase 64kB sector at 0x110000..
erase 64kB sector at 0x120000..
programming..  170788/170788
verify..       170788/170788  VERIFY OK
Bye.
```
Yes it works perfectly now thank you!

Reply:
I did include it in `__all__` as this makes it publicly exported, and maybe downstream users depend on it.

Reply:
I do not know Python best practices very well yet.
So this is where this kind of tooling is very good to have!